src/view/pages/gettingStarted.css

```diff
@@ -0,0 +1,22 @@
+.inv {
+    display: none;
+}
+.deviceSelector {

```

Comment:
It seems that we are putting line breaks between rulesets in the other CSS files.

Reply:
Added line breaks